infra/tpu-pytorch-releases/artifacts.auto.tfvars

```diff
@@ -1,4 +1,4 @@
-nightly_package_version = "2.3.0"
+nightly_package_version = "2.4.0"
```

Comment:
can you update https://github.com/pytorch/xla/blob/master/setup.py#L93 too?

Reply:
done